nextflow_schema.json

```diff
@@ -764,6 +764,18 @@
                     "description": "Maximal number of contigs that are not part of any bin but treated as individual genome.",
                     "help_text": "Contigs that do not fulfill the thresholds of `--min_length_unbinned_contigs` and `--max_unbinned_contigs` are pooled for downstream analysis and reporting, except contigs that also do not fullfill `--min_contig_size` are not considered further."
                 },
+                "bin_min_size": {
+                    "type": "integer",
+                    "default": 0,
+                    "description": "Specify the shortest length a bin should be to retain for downstream processing (in base pairs)",
+                    "help_text": "This can speed up downstream processing by removing 'nonsense' bins (e.g. badly binned single short-contig bins, or unnaturally large bins). "
```

Comment:
I have never seen a `badly binned single short-contig bin`, is that a thing? Genuine question!

Reply:
It was @alexhbnr who I think said this to me (unless I misinterpreted). I think said concoct had a tendency of this. 

Or might have be a couple of very short contigs rather than just a single one now I think about it 🤔